### PR TITLE
Enable compression for faster download of history

### PR DESCRIPTION
**Pull request by sharkoz**
This PR enables compression on nginx, mostly for the big json files containing the history of tracks. On my machine, the history for one month is 5 Mo and as my server's upload speed is pretty slow, it was taking a very long time to load. This doesn't make it perfect but it still is way faster.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,11 +2,9 @@
 
 > A modern web interface for OwnTracks made with Vue.js
 
-![Docker Pulls](https://img.shields.io/docker/pulls/owntracks/frontend)
+[![Docker Pulls](https://img.shields.io/docker/pulls/owntracks/frontend)](https://hub.docker.com/r/owntracks/frontend)
 
-<p style="text-align: center;">
-  <img src="docs/images/owntracks-ui.png" alt="OwnTracks UI">
-</p>
+![OwnTracks UI](docs/images/owntracks-ui.png)
```
Comment (linusg) [inline README.md]: I was deliberately using a bit of HTML here for `text-align: center;` :wink:

Comment (sharkoz) [reply README.md]: Sorry I rolled it back, I hoped it would fix the image not displaying on docker hub but it didn't 😞

Comment (linusg) [reply README.md]: Fair point. That's most likely caused by the relative path `docs/images/owntracks-ui.png` which GitHub handles correctly but Docker Hub probably doesn't.

Comment (linusg) [reply README.md]: Let's see if https://github.com/owntracks/frontend/commit/ed2118339da6a88daeb1eb6c4be5f8efeb5485d0 will fix this.

Comment (linusg) [reply README.md]: Hooray!

image

**Pull request merged**